src/lib/testFlags.ts

```diff
@@ -60,6 +60,10 @@ class TestFlags {
     return !!this.queryParams.cancelall;
   }
 
+  get showLimitClose() {
+    return !!this.queryParams.limitclose;
+  }
```

Comment:
i originally have statsig enabled for dev, but then i feel like that'll cause more confusion since the feature doesn't work yet for anyone. so i think i'll have the test flag at least until protocol supports it in dev/staging?